src/main/java/org/sonarsource/sonarlint/core/tracking/CachingIssueTracker.java

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Is this package *.sonarlint.core.tracking* supposed to be moved later to sonarlint-core? If not, it's better to name it something else or otherwise stack traces will be misleading.